pipeline/gene_to_nissl.py

```diff
@@ -136,12 +145,15 @@ def registration(
 
             warped_expression.append(warped)
 
+        section_numbers_kept.append(True)
+
         if (i + 1) % 5 == 0:
             logger.info(f" {i + 1} / {gene_volume.shape[0]} registrations done")
 
     warped_expression = np.array(warped_expression) if warped_expression else None
+    section_numbers_kept = np.array(section_numbers_kept, dtype=bool)
```

Comment:
IMO it is not necessary to cast into `np.ndarray` and we can just return the `list[bool]`.

Reply:
Done in [ebf4aa3](https://github.com/BlueBrain/Deep-Atlas/pull/35/commits/ebf4aa3c25f0a7a001a7c1afe4df216529563ff8).